modules/command/dc/dc-offer-create-db-command.js

```diff
@@ -0,0 +1,109 @@
+const BN = require('bn.js');
+const Models = require('../../../models/index');
+
+const Command = require('../command');
+const ImportUtilities = require('../../ImportUtilities');
+
+/**
+ * Creates offer in the database
+ */
+class DCOfferCreateDbCommand extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.config = ctx.config;
+        this.graphStorage = ctx.graphStorage;
+    }
+
+    async execute(command, transaction) {
+        const {
+            importId,
+            replicationId,
+            rootHash,
+            total_escrow_time,
```

Comment:
@simonovic86 Can we please add documentation marking that these 4 parameters are optional?